docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -222,7 +222,7 @@ New patients will be started on a high dose, high potency statin (max dose of a
 If a patient experiences a side effect, they will wither have their dose cut in half – unless they are at the minimum dose already, in which case they will be given a low potency statin.  If they are not at target, they will combine these changes with addition of ezetimibe.
 
 Initialization of patients into treatment for intervention scenarios
-+++++++++++++++++++++++++++++++++++++++++++++++++
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
 The 2 intervention scenarios (guidelines + multiple pills and guidelines + single, combination pill) are initialized the same way as the BAU case.  
 The difference is that in the intervention scenarios, treatment criteria have changed – there is now a lower threshold for treatment (based on a patient’s CV risk SCORE), including caveats for DM and CKD state.
```

Comment:
Also, new patients will start treatment on a high potency, high dose statin.  In BAU, this is not necessarily true.